server/datastore/mysql/migrations/tables/20230918132351_AddResolvedInVersionToSoftwareCVE.go

```diff
@@ -0,0 +1,36 @@
+package tables
+
+import (
+	"database/sql"
+	"fmt"
+)
+
+func init() {
+	MigrationClient.AddMigration(Up_20230918132351, Down_20230918132351)
+}
+
+func Up_20230918132351(tx *sql.Tx) error {
+	stmt := `
+		-- add resolved_in_version column to software_cve
+		ALTER TABLE software_cve
+		ADD COLUMN resolved_in_version VARCHAR(255) COLLATE utf8mb4_unicode_ci DEFAULT NULL;
+
+		-- drop the existing unique constraint
+		ALTER TABLE software_cve
+		DROP INDEX unq_software_id_cve;
+
+		-- add a new unique constraint
+		ALTER TABLE software_cve
+		ADD CONSTRAINT unq_software_id_cve UNIQUE KEY (software_id, cve, source, resolved_in_version);
```

Comment:
Why are we adding `source` and `resolved_in_version ` as part of the unique key?

Reply:
sorry, i mis-read a previous comment, i reverted this.